pkg/logs/input/docker/tailer.go

```diff
@@ -58,14 +57,14 @@ type Tailer struct {
 }
 
 // NewTailer returns a new Tailer
-func NewTailer(cli *client.Client, containerID string, source *config.LogSource, outputChan chan *message.Message, erroredContainerID chan string) *Tailer {
+func NewTailer(dockeruti *dockerutil.DockerUtil, containerID string, source *config.LogSource, outputChan chan *message.Message, erroredContainerID chan string) *Tailer {
 	return &Tailer{
 		ContainerID:        containerID,
 		outputChan:         outputChan,
 		decoder:            InitializeDecoder(source, containerID),
 		source:             source,
 		tagProvider:        tag.NewProvider(dockerutil.ContainerIDToTaggerEntityName(containerID)),
-		cli:                cli,
+		dockerutil:         dockeruti,
```

Comment:
not `dockeruti` -> `dockerutil`

Reply:
I put back the old name `cli`. In this context, we cannot name the variable `dockerutil` because it would hide the `dockerutil` package.